docs/docs/authentication.md

```diff
@@ -4,98 +4,120 @@ description: Set up an authentication provider
 
 # Authentication
 
-`@redwoodjs/auth` contains both a built-in database-backed authentication system (dbAuth), as well as lightweight wrappers around popular SPA authentication libraries.
+Redwood has integrated auth end to end, from the web side to the api side.
+On the web side, the router can protect pages via the `Private` component, and even restrict access at the role-level.
+And if you'd prefer to work with the primitives, the `useAuth` hook exposes all the pieces to build the experience you want.
 
-- dbAuth _([Redwood Docs](https://redwoodjs.com/docs/auth/dbauth))_
+Likewise, the api side is locked down by default: all SDLs are generated with the `@requireAuth` directive, making sure that making something publicly available is something that you opt into rather than out of.
+You can also require auth anywhere in your Services, and even in your server-ful or -less functions.
```

Comment:
```suggestion
You can also require auth anywhere in your Services, and even in your serverful or serverless functions.
```

What you had wasn't technically incorrect, but personally I think it reads better to just write both words out in this case